--- lib/color-picker.component.ts	2019-02-28 10:08:38.000000000 +0100
+++ lib/color-picker.component.ts	2019-02-28 10:15:12.000000000 +0100
@@ -11,8 +11,8 @@
 
 @Component({
   selector: 'color-picker',
-  templateUrl: '../../dist/lib/color-picker.component.html',
-  styleUrls: [ '../../dist/lib/color-picker.component.css' ],
+  templateUrl: 'color-picker.component.html',
+  styleUrls: [ 'color-picker.component.css' ],
   encapsulation: ViewEncapsulation.None
 })
 export class ColorPickerComponent implements OnInit, OnDestroy, AfterViewInit {
